tips/TIP-0028/event-api.yml

```
@@ -0,0 +1,1086 @@
+          type: array
+          description: Unlock condtions that define how the output an be unlocked in a transaction.
+          items:
+            anyOf:
+              - $ref: '#/components/schemas/ImmutableAliasAddressUnlockCondition'
+        features:
+          type: array
+          description: Features that add utility to the output but do not impose unlocking conditions.
+          items:
+            anyOf:
+              - $ref: '#/components/schemas/MetadataFeature'
+        immutableFeatures:
+          type: array
+          description: Immutable features that add utility to the output but do not impose unlocking conditions.
+            These blocks need to be kept in future transitions of the UTXO state machine.
+          items:
+            anyOf:
+              - $ref: '#/components/schemas/MetadataFeature'
+      required:
+        - type
+        - amount
+        - serialNumber
+        - tokenScheme
+        - unlockConditions
+    NFTOutput:
+      description: escribes an NFT output, a globally unique token with metadata attached.
+      properties:
+        type:
+          type: integer
+          description: Set to value 6 to denote a NFT Output.
+        amount:
+          type: string
+          description: The amount of IOTA tokens to deposit with this output. Encoded as plain string.
+          example: "100"
+        nativeTokens:
+          type: array
+          description: Native tokens held by the otuput.
+          items:
+            anyOf:
+              - $ref: '#/components/schemas/NativeToken'
+        nftId:
+          type: string
+          description: Unique identifier of the NFT, which is the BLAKE2b-256 hash of the Output ID that created it. NFT Address = NFT Address Type || NFT ID
+          example: "0x19c82b32761fd8729a1a6c77f7c17597e4b9b01759794e52381f6a0050b0c11f"
+        unlockConditions:
+          type: array
+          description: Unlock condtions that define how the output an be unlocked in a transaction.
+          items:
+            anyOf:
+              - $ref: '#/components/schemas/AddressUnlockCondition'
+              - $ref: '#/components/schemas/StorageDepositReturnUnlockCondition'
+              - $ref: '#/components/schemas/TimelockUnlockCondition'
+              - $ref: '#/components/schemas/ExpirationUnlockCondition'
+        features:
+          type: array
+          description: Features that add utility to the output but do not impose unlocking conditions.
+          items:
+            anyOf:
+              - $ref: '#/components/schemas/SenderFeature'
+              - $ref: '#/components/schemas/IssuerFeature'
+              - $ref: '#/components/schemas/MetadataFeature'
+              - $ref: '#/components/schemas/TagFeature'
+        immutableFeatures:
+          type: array
+          description: Immutable featuressssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss that add utility to the output but do not impose unlocking conditions.

```

too many featuressss :stuck_out_tongue: 
```suggestion
          description: Immutable features that add utility to the output but do not impose unlocking conditions.

```